packages/select/src/index.ts

```diff
@@ -23,6 +23,7 @@ type SelectConfig = AsyncPromptConfig & {
 export default createPrompt<string, SelectConfig>((config, done) => {
   const [status, setStatus] = useState('pending');
   const [cursorPosition, setCursorPos] = useState(0);
+  const [firstRender, setFirstRender] = useState(true);
```

Comment:
We should `useRef` for this one so updating it to false doesn't trigger a re-renders.

Reply:
Thanks, I'll resubmit a PR.

Reply:
Would it be better if I used `useEffect` to execute only on the first render?

Reply:
Oh, I wouldn't since that wouldn't work in React; I'm trying to keep the API consistent, but I see hooks like `useEffect` aren't behaving exactly in the same way now 😅

Reply:
Since I'm not very familiar with hooks, I can't realize this feature by simply using `useRef` here, do you have any other good solution? Thanks 😄

Reply:
No worries, ref are the right solution... But there's a bug in them and they're not referentially equal as I was hoping for... 🤦🏻 I debugged it earlier this week, but didn't have enough time to find the root cause yet.

Reply:
Thanks for your work.😄